File: frontend/src/dtos/ArchivedProjectDTO.ts
export class ArchivedProjectDTO {
    id: number
    projectId: number
    userId: number
    archivedAt: string

    constructor(id: number, projectId: number, userId: number, archivedAt: string) {
        this.id = id
        this.projectId = projectId
        this.userId = userId
        this.archivedAt = archivedAt
    }
}

File: backend/src/main/java/com/grouply/backend/archived_project/dto/ArchivedProjectDTO.java
package com.grouply.backend.archived_project.dto;
import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ArchivedProjectDTO {
    private Long id;
    private Long projectId;
    private Long userId;
    private LocalDateTime archivedAt;
}
